.changeset/light-pants-smoke.md

```diff
@@ -0,0 +1,20 @@
+---
+'astro': minor
+---
+
+Adds support for external redirects `astro.config.mjs`:
```

Comment:
```suggestion
Adds support for configured external redirects:
```
May make sense to follow their docs name (https://docs.astro.build/en/guides/routing/#configured-redirects)